### ISSUE TITLE
DL Case Study: A better way to show off image 

### ISSUE THREAD

**Issue by gabrielataginya** [BraveUX/braveux#345]
<img width="1366" alt="screen shot 2018-02-08 at 10 02 50 am" src="https://user-images.githubusercontent.com/31738798/35979825-50775648-0cb7-11e8-895c-cf659382bd6a.png">

This image (and built to inspire/vaults of inspiration images) could be in higher resolution

**Comment by ekfuhrmann** [comment]
Yes and no, we do need to be consciencous of how large these images actually are, and we want to try and keep file size as slim as possible.

That said, these specific images in the Sketch file aren't really high fidelity to begin with, meaning that any further minification on my end really hurts the image quality further.

@jennabillingsley @theseangoodwin Is there anyway we can have a higher quality version of these images within sketch. When I upsace it to 2x, it's not great. I'm guessing the original symbol/layout in which these images were pulled from was rather small, and when we export 2x it shows.

**Comment by ekfuhrmann** [comment]
Duplicate of #322
